common/Datasets/direct-lake-dataset.md

```diff
@@ -28,7 +28,7 @@ Tabular Editor 2 can connect to Direct Lake semantic models, but does not have a
 
 > [!NOTE]
 > DirectLake models currently use a collation that is different from regular Power BI import semantic models. This may lead to different results when querying the model, or when referencing object names in DAX code.
-For more information please see this blog post by Kurth Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
+For more information please see this blog post by Kurt Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
```

Comment:
I still don't see the '>' in front of this line? :)

Reply:
If you insist I am happy to add it, but it works just fine without

Reply:
You're right. That was actually surprising to me as I thought all lines had to be prefixed by that. However, there's a bug in the way the link is shown in the infobox below:

<img width="611" alt="image" src="https://github.com/TabularEditor/TabularEditorDocs/assets/8976200/277c5502-1163-49ce-9d9c-4a6c20e81bbf">

Reply:
@otykier The link has been fixed